2.4/Dockerfile

```diff
@@ -38,22 +38,26 @@ RUN yum install -y epel-release && \
     # Loosen permission bits to avoid problems running container with arbitrary UID
     chmod -R a+rwx /opt/rh/mongodb24/root/var/lib/mongodb
 
-COPY run-mongod.sh /usr/local/bin/
-COPY contrib /var/lib/mongodb/
-
 # Loosen permission bits to avoid problems running container with arbitrary UID
-RUN chmod -R g+rwx /var/lib/mongodb
+RUN chmod -R g+rwx /var/lib/mongodb && touch /etc/mongod.conf && chown mongodb.root /etc/mongod.conf && chmod og+rw /etc/mongod.conf 
```

Comment:
@mfojtik /etc/mongod.conf have to be writeable because run-mongod after substitution of env. variables have to write actual config there. In openshift version it is writable too, but it is in HOME directory...

Now using fix-permission script. Please check current status...